readme.md

```diff
@@ -623,86 +689,46 @@ interface RequestCredential {
             /** optional list of DIDs of attesters trusted by this verifier */
             trustedAttesters?: string[]
 
-            /** list of credential attributes which MUST be included when submitting the credential */
+            /**
+             * list of credential attributes which MUST be included when submitting the credential.
+             * The properties are identified using JSONPath.
```

Comment:
let's do json pointers, which are simpler and more specific

Reply:
fixed in https://github.com/KILTprotocol/spec-ext-credential-api/commit/7d37be367d3ef90811deb76b644782f598834d41